randomproto.py

```diff
@@ -1,47 +1,46 @@
-"""Random protobuf message generator."""
```

Comment:
Why is the docstring removed?

Reply:
Just doesn't look useful to me. Package description is already present in setup.py

Reply:
This is the module (file) docstring, which is in general different from the package description. For this package it's true that there's no much difference in them. However I'd still prefer keeping it as a good practice. Some Python linters also require the presence of module docstrings.

Reply:
Ok, understood